presto-cassandra/src/main/java/com/facebook/presto/cassandra/NativeCassandraSession.java

```diff
@@ -171,6 +173,9 @@ public String getCaseSensitiveSchemaName(String caseInsensitiveSchemaName)
         for (TableMetadata table : keyspace.getTables()) {
             builder.add(table.getName());
         }
+        for (MaterializedViewMetadata materializedView : keyspace.getMaterializedViews()) {
```

Comment:
Also, "Currently, only simple SELECT statements are supported, but a ticket has been filed to add support for more complex SELECT statements
WHERE clauses, ORDER BY, and functions aren't available with materialized views" (https://www.datastax.com/dev/blog/new-in-cassandra-3-0-materialized-views) -- That suggests that Presto doesn't support the initial, Cassandra 3.0 version of materialized views on partitioning keys, since we push down a where clause in that case. Can you only add materialized views here if the Cassandra version is >= whatever Cassandra version adds full materialized view support? You can get the version via `getCassandraVersion()` in `CassandraSession`